### ISSUE TITLE
Narrator mode on GNU/Linux doesn't work correctly

### ISSUE THREAD

**Issue by viachaslavic** [libretro/RetroArch#15888]
## Description

It looks like RetroArch used `espeak` as an TTS on Linux. I found some issues with the current implementation.

### Expected behavior

1. [x]  To speak according to the rules of the language specified by the __AI Service → Target Language__ parameter (`MENU_ENUM_LABEL_VALUE_AI_SERVICE_TARGET_LANG`).
2. [x]  If TTS not installed, then display an On-Screen Notification with error instead of quitting

### Actual behavior

1. Narrator depends only on the __User Language__ setting , even ignoring the explicitly setting of the __AI Target Language__. In this regard TTS engine may speaks the translated string by letters instead by words.
2. If TTS not installed, subprocess terminated with the main process with exit code 1 after requesting to the AI Service.

```log
...
[NFO] [Translate]: Sending request to: <URI>
[NFO] Spoke: <translated text>
X connection to :0 broken (explicit kill or server shutdown).
X connection to :0 broken (explicit kill or server shutdown).
```

### Steps to reproduce the bug

1. Enable AI Service in Narrator-compatible mode.
2. Set UI Language different from the AI Target Language (for first case)
3. Run game and request AI Service (remove `espeak` from system for second case)

### Version/Commit

```console
$ ./retroarch --version
RetroArch - Frontend for libretro
Version: 1.16.0 (Git f20c8493b3) Nov 10 2023
Compiler: GCC (13.2.1) 64-bit
```

### Environment information

- OS:  Arch Linux  (Linux 6.5.7-zen2-1-zen x86_64 GNU/Linux)
- Compiler: gcc (GCC) 13.2.1 20230801
- X11 Window manager: dwm-6.4
- eSpeak NG text-to-speech: 1.51.1  Data at: /usr/share/espeak-ng-data

**Comment by viachaslavic** [comment]
> display an On-Screen Notification with error

Now the error is logged. I'm not sure about a on-screen notify. That may not be informative because app not linked to eSpeak explicitly.

**Comment by BarryJRowe** [comment]
This PR should not have been merged. The AI service and the Accessibility feature are not the same thing! The User Language setting is what language is spoken in the UI, Menus. The AI Service language is only intended for AI service functionality and should not be used when reading out things from the menu. Testing master now, it's reading out my english menu UI text with a French speaker because that was the last language I used for AI translation.
